apps/protocol-ld/src/subgraph.ts

```diff
@@ -0,0 +1,59 @@
+// Environment Variables.
+import { GraphQLClient } from 'graphql-request';
+import { GovrnGraphClient } from '@govrn/govrn-subgraph-client';
+import { GovrnContract, NetworkConfig } from '@govrn/govrn-contract-client';
+import { BigNumber, ethers } from 'ethers';
+import { LDContribution } from './types';
+
+const SUBGRAPH_ENDPOINT = process.env.SUBGRAPH_URL;
+const CONTRACT_ADDRESS = process.env.CONTRACT_ADDRESS;
+const CHAIN_URL = process.env.CHAIN_URL;
+
+const networkConfig: NetworkConfig = {
+  address: CONTRACT_ADDRESS,
+  chainId: 2,
+};
+
+// const sfsdf = async (contr: any) => {
+//   const detailsUri = ethers.utils.toUtf8String(contr.detailsUri);
+//   const ipfsBlob = await fetchIPFS(detailsUri);
+//   const ipfsText = await ipfsBlob.text();
+//   const contributionDetails = JSON.parse(ipfsText);
+//   return {
+//     detailsUri,
+//   };
+// };
+
+// load data from subgraph.
+const provider = new ethers.providers.JsonRpcProvider(CHAIN_URL);
+const govrnContract = new GovrnContract(networkConfig, provider);
+
+export const loadContributions: () => Promise<
+  Awaited<LDContribution>[]
+> = async () => {
+  const graphQLClient = new GraphQLClient(SUBGRAPH_ENDPOINT);
+  const client = new GovrnGraphClient(graphQLClient);
```

Comment:
This devx is kind of bad. We should probably update the subgraph client, so instead of taking in a GraphQLClient it takes in a chainId. Within the subgraph client we have a constant with a mapping of chainId to subgraph url  that we use to instantiate a GraphQLClient under the hood.